With OSX I'm not able to build. <samplerate.h> is not found with the latest rack:

`utils/WavWriter.cpp:94:3: error: use of undeclared identifier 'src_float_to_short_array'
                src_float_to_short_array(m_buffer.data()->samples, buffer.data(), sampleCount);
                ^
1 error generated.`

--- a/Makefile
+++ b/Makefile
@@ -2,13 +2,26 @@ SLUG = IO-Simple
 VERSION = 0.6.0dev
 
 FLAGS += -I"."
-LDFLAGS += -lsamplerate
+
+RACK_DIR ?= ../..
+include $(RACK_DIR)/arch.mk
+
+ifeq ($(ARCH), win)
+LDFLAGS += /mingw64/lib/libsamplerate.a
+endif
+
+ifeq ($(ARCH), mac)
+LDFLAGS += $(shell pkg-config --variable=libdir samplerate)/libsamplerate.a
+endif
+
+ifeq ($(ARCH), lin)
+LDFLAGS += $(shell pkg-config --variable=libdir samplerate)/libsamplerate.a
+endif
 
 SOURCES += $(wildcard src/*.cpp utils/*.cpp utils/*.c)
 
 DISTRIBUTABLES += $(wildcard LICENSE*) res
 
-RACK_DIR ?= ../..
 include $(RACK_DIR)/plugin.mk
 
 serve:

